MSAL/test/integration/native_auth/end_to_end/otp_code_retriever/MSALNativeAuthEmailCodeRetriever.swift

```diff
@@ -76,7 +84,19 @@ class MSALNativeAuthEmailCodeRetriever: XCTestCase {
             }
             if dataDictionary.count > 0 {
                 dataDictionary.sort(by: {($0["id"] as? Int ?? 0) > ($1["id"] as? Int ?? 0)})
```

Comment:
Should we change the sort method type? Instead of using the id we should use date, so we always check the latest email. Can you confirm?

Reply:
Tested again now, higher id's have newer dates so the sorting is the same. We would need to convert the string to date to be able to compare them which would make the calculation more intensive.